### PR TITLE
sdl_rc_baseline_part2

### PR DESCRIPTION
**Pull request by OlesiaV**
Guidelines for ButtonPress and SetInteriorVehicledata

### CODE REVIEW

File: docs/Common/Enums/index.md [hunk 1]
```diff
@@ -336,6 +336,26 @@
 |PRESET_9|14||
 |CUSTOM_BUTTON|15||
 |SEARCH|16||
+Climate Buttons
+|AC_MAX|17||
+|AC 18|||
```
Comment (yang1070) [inline docs/Common/Enums/index.md]: shall be |AC | 18||

File: docs/RC/SetInteriorVehicleData/index.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+|moduleData|[Common.ModuleData](https://github.com/OlesiaV/sdl_hmi_integration_guidelines/blob/a118768fa9c6631e168f932919687196bd281f46/docs/Common/Structs/index.md)|true|||
+
+### Sequence Diagrams
+|||
+SetInteriorVehicleData
+![SetInteriorVehicleData](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/SDL_RC_baseline_part2/docs/RC/SetInteriorVehicleData/accessories/SetInteriorVehicleData.png)
+|||
+|||
+SetInteriorvehicleData READ_ONLY
+![SetInteriorvehicleData READ_ONLY](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/SDL_RC_baseline_part2/docs/RC/SetInteriorVehicleData/accessories/setInteriorVehicleData_READ_ONLY.png)
```
Comment (yang1070) [inline docs/RC/SetInteriorVehicleData/index.md]: in the image file, please remove "device of any rank (driver's or passenger's ...)

File: docs/Common/Enums/index.md (hunk 1 above)
Comment (OlesiaV) [inline docs/Common/Enums/index.md]: corrected

File: docs/RC/SetInteriorVehicleData/index.md (hunk 2 above)
Comment (OlesiaV) [inline docs/RC/SetInteriorVehicleData/index.md]: file changed

**Comment by JackLivio** [comment]
@OlesiaV @LuxoftAKutsan Please add new files to config.json so the new pages can be navigated to on developer portal. Also ButtonPress RPC should go in RC directory, not in the button interface directory.

**Comment by LuxoftAKutsan** [comment]
@JackLivio @yang1070 changes from this PR are available in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/69

**Comment by LuxoftAKutsan** [comment]
@JackLivio Button press moved to RC in #69

**Pull request closed without merging**